sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_models.py

```diff
@@ -438,3 +438,13 @@ def scheduled_purge_date(self):
         :rtype: ~datetime.datetime
         """
         return self._scheduled_purge_date
+
+
+class RandomBytes(object):
+    """Contains random bytes returned from :func:`~azure.keyvault.keys.KeyClient.get_random_bytes`.
```

Comment:
Having overthought this, I think we should either not have a link back to the client, or add a link to the async method:
- I expect most readers to come from this link i.e. reading the method docs, they want to see details of the model
- someone working on an async application may follow this to the sync client's docs
    - probably they will survive but there's a potential for confusion around async support

A second link would require fully qualified text. That isn't a guaranteed win and makes the whole thing harder to read. So, I guess I would lean toward replacing the link with "a Managed HSM", but really I'm just trying to provide food for thought here 😇